packages/snap/src/index.ts

```diff
@@ -1,31 +1,23 @@
 import { OnRpcRequestHandler } from '@metamask/snap-types';
-import {
-  InitOutput,
-  // add_random,
-  add_random_with_seed
-} from 'wasm-bundler';
-import { getRandomBytes } from './random';
 
-import { initializeWasm } from './wasm';
+import initWasm from '../../../rust-rng/dist/index';
+
+let wasm;
 
-let wasm: InitOutput;
 
 export const onRpcRequest: OnRpcRequestHandler = async ({
   origin,
   request,
 }) => {
   if (!wasm) {
-    wasm = await initializeWasm();
+    wasm = await initWasm();
```

Comment:
When using `smaldot` we don't have to manually init the WASM module like this. It tells me that we're not doing yet what they are doing. Maybe there is some room for further refinement.